universe/supplycommit/manager.go

```diff
@@ -273,6 +280,125 @@ func (m *Manager) SendEvent(ctx context.Context,
 	return nil
 }
 
+// IgnoreAssetOutPoint allows an asset issuer to mark a specific asset outpoint
+// as ignored. An ignored outpoint will be included in the next universe
+// commitment transaction that is published.
+func (m *Manager) IgnoreAssetOutPoint(ctx context.Context,
+	assetSpec asset.Specifier, assetAnchorPoint asset.AnchorPoint,
+	amount uint64) (universe.SignedIgnoreTuple, error) {
+
+	var zero universe.SignedIgnoreTuple
+
+	assetID, err := assetSpec.UnwrapIdOrErr()
+	if err != nil {
+		return zero, err
+	}
+
+	// Fetch the asset group for the given asset ID. If a group key was
+	// provided, use it for verification; otherwise, fall back to this
+	// group key.
+	assetGroup, err := m.cfg.AssetLookup.QueryAssetGroupByID(ctx, assetID)
+	if err != nil {
+		return zero, fmt.Errorf("fail to find asset group given "+
+			"asset ID: %w", err)
+	}
+
+	// If the asset specifier includes a group key, ensure it matches the
+	// group key of the asset group fetched from the database.
+	if assetSpec.HasGroupPubKey() {
```

Comment:
At present I think the new method is only ever called with an assetSpec constructed from asset.NewSpecifierFromId, so this block feels strange -- assetSpec.HasGroupPubKey will always be false.